test/ffmpeg-d3d11/decode/hevc.py

```diff
@@ -21,5 +21,7 @@ def before(self):
   @slash.parametrize(("case"), sorted(spec.keys()))
   def test(self, case):
     vars(self).update(spec[case].copy())
+    if spec[case].get('features', {}) and spec[case].get('features', {})[0] == 'scc':
```

Comment:
The "features" is a list.  Thus, default return value should be a list `[]`, not a dict `{}`.
How about simplify this to:
```python
if "scc" in vars(self).get("features", list()):
  ...
```